readme.md

```diff
@@ -143,14 +163,14 @@ My first stab at the user stories for the game of concentration:
 - If two cards are face-up, and they match, the user leaves them face-up
 - If two cards are face-up, and they don't match, the user turns them face-down
```

Comment:
These aren't user stories. If they were, they'd look like:

> As a user, I want to see all cards starting face down, so that I can pick from any card.
> As a user, I want to click on a card and see the front, so that I can look for matching cards.
> As a user, I want to flip over a second card, so the I can try to get a match with the first.
> As a user, I want to see my matches remain flipped face up, so that I can focus on the remaining cards.